sharing/index.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2020-01-13 17:00:00
+title: Sharing GIS Data
+categories: []
+---
+
+#### Quick Links
+
+[Submitting Data to the SGID]({% link sharing/process/index.md %}) — [Basic Metadata Creation Process]({% link sharing/metadata/index.md %}) — [AGRC Sharing Policies]({% link about/policy/index.md %})
+
+## The Joys of Sharing
+
+The Utah [State Geographic Information Database (SGID)]({% link sgid/index.html%}) is one of the premier statewide GIS databases in the country, providing clean and consistent authoritative data across every geography in the state. People have been turning to the SGID for nearly 30 years to support their personal, commercial, academic, and government projects.
```

Comment:
```suggestion
The Utah [State Geographic Information Database (SGID)]({% link sgid/index.html%}) is one of the premier statewide GIS data resources in the country, providing clean and consistent authoritative data across every geography in the state. People have been turning to the SGID for nearly 30 years to support their personal, commercial, academic, and government projects.
```
Since SGID is distributed among the three primary places, and no longer just a database ;)

Reply:
I'll implement this manually (another suggestion made this one outdated)